src/scripts/blah_common_submit_functions.sh

```diff
@@ -130,6 +130,34 @@ function bls_fl_subst_and_accumulate ()
   done
 }
 
+function bls_fl_test_exists ()
+{
+#
+# Usage: bls_fl_test_exists container_name
+# Verfies all container_name "@@F_LOCAL" exists
+# First missing file is returned in $bls_fl_test_exists_result.
+#
+  local container_name
+
+  container_name=${1:?"Missing container name argument to bls_fl_subst_and_accumulate"}
+
+  local last_argument
+
+  eval "last_argument=\${bls_${container_name}_counter:=0}"
+
+  local ind
+  bls_fl_test_exists_result=
+  for (( ind=0 ; ind < $last_argument ; ind++ )) ; do
+      bls_fl_subst $container_name $ind "@@F_LOCAL"
+      if [ ! -z "$bls_fl_subst_result" -a ! -f "$bls_fl_subst_result" ] ; then
```

Comment:
I'm not super familiar with this, but for the `! -z "$bls_fl_subst_result"` part of this condition -- under what circumstances can `$bls_fl_subst_result` be empty?  My read of `bls_fl_subst` and `bls_fl_add_value` is that `local_file_name`/`remote_file_name` always get specified together, and an empty `local_file_name` is not allowed in `bls_fl_add_value`.

I'm not really objecting, because `bls_fl_subst`, which you're calling, makes it seem like it will return an empty `$bls_fl_subst_result` if `$f_local` or `$f_remote` is empty, it's just odd to me because it looks like that condition (either being empty) can't ever happen.  Ah well.